Rock.Rest/Controllers/FinancialTransactionsController.Partial.cs

```diff
@@ -317,7 +317,33 @@ public DataSet GetContributionTransactions( int groupId, int? personId, [FromBod
         }
 
         /// <summary>
-        /// 
+        /// Gets the contribution transactions.
+        /// </summary>
+        /// <param name="groupId">The group unique identifier.</param>
+        /// <returns></returns>
+        /// <exception cref="System.Web.Http.HttpResponseException"></exception>
+        [Authenticate, Secured]
+        [HttpGet]
+        [EnableQuery]
+        [System.Web.Http.Route( "api/FinancialTransactions/GetTransactionsForGivingGroup/{personId}" )]
+        public IQueryable<FinancialTransaction> GetTransactionsForGivingGroup( int personId )
+        {
```

Comment:
@benwiley @azturner the `GetTransactionsForGivingGroup` feature has not been shipped yet. If `GetByGivingId( int givingId )` is an easier and more performant method, I can start to send the `GivingId` to the client for this call.